README.md

```diff
@@ -119,13 +131,48 @@ By default, the loader initializes a system with one core and 512 MiB RAM.
 
 **Example:** the following command starts the demo application in a virtual machine, which has 4 cores and 8GiB memory:
 
-```bash
+```sh
 HERMIT_CPUS=4 HERMIT_MEM=8G uhyve /path/to/the/unikernel/binary
 ```
 
-## Debugging of Hermit apps (unstable)
+### Known issues
+
+ * Uhyve isn't able to pass more than 128 environment variables to the unikernel.
+
+## Licensing
+
+Licensed under either of
+
+ * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
+ * MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
+
+at your option.
+
+Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any additional terms or conditions.
+
+---
+
+## Contributing
+
+### Testing Hermit apps using `cargo run`
+
+As mentioned above, the Uhyve repository ships some binaries that can be used for testing purposes.
+
+If you are hacking on Uhyve's source code, you can also use `cargo run` to immediately test your changes:
+
+```sh
+cargo run -- -v data/x86_64/rusty_demo
+cargo run -- -v data/x86_64/hello_world
+cargo run -- -v data/x86_64/hello_c
+```
+
+### Debugging Hermit apps
+
+> [!WARNING]
+> This feature is unstable.
```

Comment:
Final nitpicks: Please remove this, this feature is as unstable as everything else :grin: